cloudigrade/api/tests/test_models.py

```diff
@@ -193,3 +194,33 @@ def test_synthetic_data_request_post_delete_callback(self):
         self.assertTrue(CloudAccount.objects.filter(user_id=user_id).exists())
         self.request.delete()
         self.assertFalse(CloudAccount.objects.filter(user_id=user_id).exists())
+
+
+class UserModelTest(TestCase):
+    """UserModel tests not covered indirectly via other tests."""
+
+    def test_user_object_string_representation(self):
+        """Test that the string includes the uuid, account_number and org_id."""
+        user_uuid = str(uuid.uuid4())
+        user_account_number = "11111111"
+        user_org_id = "22222222"
```

Comment:
Let's `faker` these values.

Reply:
I had the faker calls earlier, but then assertion would fail because it saw Faker objects instead of the strings.  I'll try again.

Reply:
Weird… most faker functions should always return common primitives like `int` and `str`. If faker returns an `int`, it would be fine to wrap that in `str(...)` to make any of our type-checking happy.

Reply:
str() to the rescue 👍